cypress/tools/fixture-generator.py

```diff
@@ -0,0 +1,54 @@
+import json
+import random
+import string
+import uuid
+from datetime import datetime, timedelta
+import os
+
+def random_string(length=8):
+    return ''.join(random.choices(string.printable + 
+                                  string.digits + 
+                                  string.punctuation + 
+                                  string.whitespace, k=length))
+
+def random_date(start, end):
+    delta = end - start
+    random_days = random.randint(0, delta.days)
+    return start + timedelta(days=random_days)
+
+start_date = datetime(2020, 1, 1)
+end_date = datetime(2025, 1, 1)
+
+data = []
+
+for _ in range(15):
```

Comment:
Let's produce much larger number of templates. This will allow for greater variety, which might help catch more possible edge cases. And we might want to be sure that our app is working fine even with a large number of templates on the backend. How about 100-150?